e2e/test_operator_ha/operator_ha_test.go

```diff
@@ -211,4 +213,109 @@ var _ = Describe("Operator HA tests", Label("e2e", "pr"), func() {
 			}).WithTimeout(10 * time.Minute).WithPolling(2 * time.Second).Should(BeNumerically(">=", desiredRunningPods))
 		})
 	})
+
+	When("locality based exclusions are enabled", func() {
+		var initialUseLocalitiesForExclusion bool
+
+		BeforeEach(func() {
+			spec := fdbCluster.GetRemote().GetCluster().Spec.DeepCopy()
+			initialUseLocalitiesForExclusion = fdbCluster.GetRemote().GetCluster().UseLocalitiesForExclusion()
+			spec.AutomationOptions.UseLocalitiesForExclusion = pointer.Bool(true)
+			fdbCluster.GetRemote().UpdateClusterSpecWithSpec(spec)
+			Expect(fdbCluster.GetRemote().GetCluster().UseLocalitiesForExclusion()).To(BeTrue())
+		})
+
+		AfterEach(func() {
+			spec := fdbCluster.GetRemote().GetCluster().Spec.DeepCopy()
+			spec.AutomationOptions.UseLocalitiesForExclusion = pointer.Bool(initialUseLocalitiesForExclusion)
+			fdbCluster.GetRemote().UpdateClusterSpecWithSpec(spec)
+		})
+
+		When("when a remote log has network latency issues and gets replaced", func() {
+			var experiment *fixtures.ChaosMeshExperiment
+
+			BeforeEach(func() {
+				spec := fdbCluster.GetRemote().GetCluster().Spec.DeepCopy()
+				spec.AutomationOptions.UseLocalitiesForExclusion = pointer.Bool(true)
+				fdbCluster.GetRemote().UpdateClusterSpecWithSpec(spec)
+				Expect(fdbCluster.GetRemote().GetCluster().UseLocalitiesForExclusion()).To(BeTrue())
```

Comment:
seems duplicated from the beforeEach above?

Reply:
Good catch, I copied that around and missed that one 👍